include/sound/soc.h

```diff
@@ -646,6 +646,11 @@ struct snd_soc_dai_link_component {
 	const char *dai_name;
 };
 
+struct snd_soc_dai_link_codec_ch_map {
+	unsigned int connected_cpu_id;
```

Comment:
what does 'ID' mean? the dai->id or the index in the dailink?

Also if we want to support N:M, is there not a case where the same codec dai is connected to multiple cpu DAIs?

Or is the plan to support N:M with N < M ?

Reply:
> what does 'ID' mean? the dai->id or the index in the dailink?

The 'ID' means the dai->id. i.e. The index of the `cpus`. 

> 
> Also if we want to support N:M, is there not a case where the same codec dai is connected to multiple cpu DAIs?
> 

hmm, a codec can only connect to a sdw link physically, right? That's why I didn't think about the same codec dai is connected to multiple cpu DAIs case.

> Or is the plan to support N:M with N < M ?

Yes.